pkg/metadata/handlers.go

```diff
@@ -17,36 +17,57 @@ limitations under the License.
 package metadata
 
 import (
+	"encoding/json"
 	"errors"
 	"fmt"
 	"net/http"
 
 	"github.com/gorilla/mux"
 	"github.com/redis/go-redis/v9"
+	"github.com/vllm-project/aibrix/pkg/cache"
 	"github.com/vllm-project/aibrix/pkg/utils"
 	"k8s.io/klog/v2"
 )
 
 type httpServer struct {
 	redisClient *redis.Client
+	cache       *cache.Cache
 }
 
 func NewHTTPServer(addr string, redis *redis.Client) *http.Server {
+	c, err := cache.GetCache()
+	if err != nil {
+		panic(err)
+	}
+
 	server := &httpServer{
 		redisClient: redis,
+		cache:       c,
 	}
 	r := mux.NewRouter()
 	r.HandleFunc("/CreateUser", server.createUser).Methods("POST")
 	r.HandleFunc("/ReadUser", server.readUser).Methods("POST")
 	r.HandleFunc("/UpdateUser", server.updateUser).Methods("POST")
 	r.HandleFunc("/DeleteUser", server.deleteUser).Methods("POST")
+	r.HandleFunc("/v1/models", server.models).Methods("GET")
 
 	return &http.Server{
 		Addr:    addr,
 		Handler: r,
 	}
 }
 
+// models returns base and lora adapters registered to aibrix control plane
+func (s *httpServer) models(w http.ResponseWriter, r *http.Request) {
+	models := s.cache.GetModels()
+	jsonBytes, err := json.Marshal(models)
```

Comment:
Let's make this compatible with vLLM's current implementation.

here's one example https://aibrix.readthedocs.io/latest/features/runtime.html#model-configuration-api

Reply:
No reservations in following open ai format in response as well. But retrieving model information from pod is tricky as api_key is not available at gateway.

Reply:
We do not necessary need to fetch from pods, just keep the structure same might be helpful I think. value could be empty.

Reply:
In this case, user can not use sdk to get the response. We can not assume user just use curl